test/data/css/tests.js

```diff
@@ -230,6 +230,19 @@ exports.test_data = {
             { input: '#bla, #foo{\n\tcolor:black;{{separator_input}}\tfont-size: 12px;\n}', output: '#bla,\n#foo {\n\tcolor: black;{{separator_output}}\tfont-size: 12px;\n}' }
         ],
     }, {
+        name: "Preserve Newlines and newline_between_rules",
+        description: "",
+        options: [
+            { name: "preserve_newlines", value: "true" },
+            { name: "newline_between_rules", value: "true" },
+        ],
+        tests: [
+            { input: '.div {}.span {}', output: '.div {}\n\n.span {}' },
```

Comment:
Take these tests and move them into the previous section. 
Then change the matrix options above to  the 4 combinations of `preserve_newlines` and `newline_between_rules`.